src/Native/System.Native/pal_networking.h

```diff
@@ -58,6 +58,12 @@ enum AddressFamily : int32_t
     PAL_AF_INET6 = 23, // System.Net.AddressFamily.InterNetworkV6
 };
 
+enum MulticastOption : int32_t
+{
+    PAL_MULTICAST_ADD = 0, // IP{,V6}_ADD_MEMBERSHIP
```

Comment:
Do these values map directly to native values? If so, please static_assert that they do